engine/common/src/main/java/org/enso/common/RuntimeOptions.java

```diff
@@ -124,19 +123,13 @@ public class RuntimeOptions {
 
   public static final String ENABLE_EXECUTION_TIMER = optionName("enableExecutionTimer");
 
-  @Option(
```

Comment:
Why are these annotations with _seemingly helpful descriptions_ removed but options themselves kept?

Reply:
I concluded (to myself) that nobody can be using the `@Option` annotation values. No test failed anywhere.

Reply:
I see. Sounds fair to remove them, although I'd prefer to keep the `help` text as a javadoc comment for one of the fields - I find these texts quite helpful in understanding what each option stands for.

Reply:
I can keep them. a238f0ec59. Somebody should however write documentation for the other options.